client/client.go

```diff
@@ -200,7 +200,11 @@ func (client *Client) ServerVersion() (string, error) {
 		version = "unknown"
 	}
 
-	return fmt.Sprintf("%s (series %s)", version, sysInfo.Series), nil
+	if sysInfo.OnClassic {
+		return fmt.Sprintf("%s (series %s, os: %s, classic)", version, sysInfo.Series, sysInfo.OS), nil
```

Comment:
Yes, that's probably the best option. If all we have is snapd and nothing else, then we don't have to show anything else.